### DISCUSSION TITLE
Demo?

### DISCUSSION THREAD

**Discussion by faizananwerali** [awcodes/filament-tiptap-editor#131] [Q&A]
Is there any demo for it?

You have shared the screenshot on the repo readme. I want the same functionality, can you share the demo from where you took the screenshot?

**Comment by awcodes** [comment]
There's no demo. The screenshot is just the default field. Nothing special about it.

**Reply by pboivin** [reply]
FYI, I'll integrate TipTap in the [Peek demo repo](https://github.com/pboivin/filament-peek-demo) for v1.0. Two birds... :)
